Makefile

```diff
@@ -3,6 +3,9 @@ GO_BUILD_VER=v0.49
 
 SEMAPHORE_PROJECT_ID?=$(SEMAPHORE_CALICOCTL_PROJECT_ID)
 
+KUBE_MASTER_PORT?=8080
```

Comment:
Following Kubernetes lead, we're trying to move away from using terms like master/slave, suggest swapping this out for `KUBE_API_PORT`